Library/Homebrew/formula_installer.rb

```diff
@@ -786,7 +791,7 @@ def pour
       path.extend(InstallRenamed)
       path.cp_path_sub(formula.bottle_prefix, HOMEBREW_PREFIX)
     end
-    FileUtils.rm_rf formula.bottle_prefix
+    FileUtils.rm_rf Dir["#{formula.bottle_prefix}/{etc,var}"]
```

Comment:
With this change, line 794 now parallels line 790. The two directories `etc` and `var` are installed and then removed from `.bottle`.
https://github.com/sjackman/brew/blob/dd27921d6a096fc7f6bbcf8f04786cb669283d95/Library/Homebrew/formula_installer.rb#L790-L794
```ruby
    Pathname.glob("#{formula.bottle_prefix}/{etc,var}/**/*") do |path|
…
    end
    FileUtils.rm_rf Dir["#{formula.bottle_prefix}/{etc,var}"]
```

My other suggestion would be `FORMULA/VERSION/.homebrew/FORMULA.rb`